src/platform/utilities/cerner/index.js

```diff
@@ -6,10 +6,14 @@ export const isCernerLive = !environment.isProduction();
 
 export const getCernerURL = path => {
   const root = environment.isProduction()
-    ? 'https://patientportal.myhealth.va.gov'
-    : 'https://ehrm-va-test.patientportal.us.healtheintent.com';
+    ? `https://patientportal.myhealth.va.gov/clear-session?to=${encodeURIComponent(
```

Comment:
This would be my suggestion:

```
const host = environment.isProduction()
  ? 'https://patientportal.myhealth.va.gov'
  : 'https://ehrm-va-test.patientportal.us.healtheintent.com';

return `${host}/clear-session?to=${encodeURIComponent(`${host}${path}`)}`;
```

Reply:
Oooo I like that much more, updating

Reply:
Thanks! 🙂